Makefile changes for ONIE patch

Copyright (C) 2017 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

Minor Makefile tweaks to work with the ONIE build system.

diff --git a/Make.defaults b/Make.defaults
index aa6a690..3349161 100644
--- a/Make.defaults
+++ b/Make.defaults
@@ -20,7 +20,7 @@ CROSS_COMPILE	?=
 PKG_CONFIG = $(CROSS_COMPILE)pkg-config
 CC	:= $(if $(filter default,$(origin CC)),$(CROSS_COMPILE)gcc,$(CC))
 CCLD	:= $(if $(filter undefined,$(origin CCLD)),$(CC),$(CCLD))
-CFLAGS	?= -O2 -g -flto
+CFLAGS	+= -O2 -g -flto
 AR	:= $(CROSS_COMPILE)gcc-ar
 NM	:= $(CROSS_COMPILE)gcc-nm
 RANLIB	:= $(CROSS_COMPILE)gcc-ranlib
